google-cloud-speech/lib/google/cloud/speech/audio.rb

```diff
@@ -260,15 +265,15 @@ def recognize max_alternatives: nil, profanity_filter: nil, phrases: nil
         #   results = op.results
         #
         def process max_alternatives: nil, profanity_filter: nil,
-                    phrases: nil
+                    phrases: nil, word_times: nil
```

Comment:
I strongly feel that this parameter should be named `words` (rather than `word_times`), because it directs the service to return "a list of words". (I know this is a departure from my usual philosophy of maintaining continuity with the underlying API(s), but in this case I believe `words` is a much clearer name.)